### PR TITLE
[FSPE-5294] Added 0.5s delay to Tooltip animation

### PR DESCRIPTION
**Pull request by janibolkvadze**
https://ctl.atlassian.net/browse/FSPE-5294

@dani-cl-madrid @SergioQCL @mattnickles @jllr

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1091/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://2a6e5a3294353487ef45dada9e03a2ca5740f83f-chi-pr.enterprise.ctl.io" target="_blank"> https://2a6e5a3294353487ef45dada9e03a2ca5740f83f-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1092/ for more details.

File: src/chi/javascript/components/tooltip.js [hunk 1]
```diff
@@ -30,13 +31,17 @@ class Tooltip extends Component {
...
+    this._addEventHandler(this._elem, 'mouseenter', () => {
       self._hovered = true;
-      if (!self._shown) {
-        self.show();
-      }
+      this._animationTimeout = window.setTimeout(() => {
+        if (!self._shown) {
+          self.show();
+        }
+      }, 500);
```
Comment (dani-cl-madrid) [inline src/chi/javascript/components/tooltip.js:40]: Please, use a constant for this. Everything else seems to be perfect.

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1099/ for more details.

**Comment by janibolkvadze** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://ed2d6a9628fe3a7e983354f9b52d9a08979712a4-chi-pr.enterprise.ctl.io" target="_blank"> https://ed2d6a9628fe3a7e983354f9b52d9a08979712a4-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1100/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-wqbnla, https://ops.elasticbox.com/services/instances/i-m51sgw

**Pull request merged**